docs/root/intro/version_history.rst

```diff
@@ -48,6 +48,7 @@ Version history
 * redis: added :ref:`latency stats <config_network_filters_redis_proxy_per_command_stats>` for commands.
 * redis: added :ref:`success and error stats <config_network_filters_redis_proxy_per_command_stats>` for commands.
 * redis: migrate hash function for host selection to `MurmurHash2 <https://sites.google.com/site/murmurhash>`_ from std::hash. MurmurHash2 is compatible with std::hash in GNU libstdc++ 3.4.20 or above. This is typically the case when compiled on Linux and not macOS.
+* redis: added support for Redis cluster custom cluster type.
```

Comment:
Please merge master and move this into the 1.11.0 section. Also please ref link to the docs.